src/Validator/Integer.php

```diff
@@ -69,6 +84,12 @@ public function getType(): string
      */
     public function isValid($value)
     {
+        if($this->loose) {
+            if(!\is_numeric($value)) {
+                return false;
+            }
+            $value = $value+0;
```

Comment:
Same as mentioned with the Float validator, can you check if `intval()` might be better?

Reply:
These I researched before as well for Range validator, you can check I use similar approach there.

It's because `intval('1.2')` would make it `1` and cause validator to accept float, similarly `floatval('1')` will make it `1.0` and cause validator to accept integer as well